src/bebb/entity.mo

```diff
@@ -147,6 +179,11 @@ module {
      * Contains all the bridge ids that point to this entity
     */
     toIds : EntityAttachedBridges;
+
+    /**
+     * Stores all the previews that are available for the current entity
+    */
+    previews : [EntityPreview];
```

Comment:
I think this is good for now. Going forward, we could think about making this e.g. a HashMap that lets us identify previews by their key

Reply:
yea true. I guess since all of them are returned anyway it doesn't really matter. In the future we can maybe just retrieve one of the previews.

Reply:
Honestly we probably want to do that in 0.1.0.